server/db/db.js

```diff
@@ -1,4 +1,5 @@
 const { CHATBOT_STATE, helpers } = require('brave-alert-lib')
+const { runQuery } = require('brave-alert-lib/lib/helpers')
```

Comment:
`helpers` has already been required above. So you can just use `helpers.runQuery` anywhere in the file.